sys-devel/dwz/dwz-0.13.ebuild

```diff
@@ -5,25 +5,21 @@ EAPI=7
 
 inherit toolchain-funcs
 
-FPKG_HASH="1624afa75b94633e03c6e1bb952fb348"
-
 DESCRIPTION="DWARF optimization and duplicate removal tool"
-HOMEPAGE="https://sourceware.org/git/?p=dwz.git;a=summary"
-SRC_URI="https://src.fedoraproject.org/repo/pkgs/dwz/${P}.tar.bz2/${FPKG_HASH}/${P}.tar.bz2"
+HOMEPAGE="https://sourceware.org/dwz"
+SRC_URI="https://sourceware.org/ftp/dwz/releases/${P}.tar.xz"
+
+S="${WORKDIR}/${PN}"
 
 LICENSE="GPL-2+ GPL-3+"
 SLOT="0"
 KEYWORDS="~amd64"
-IUSE=""
 
 DEPEND="dev-libs/elfutils"
 RDEPEND="${DEPEND}"
-BDEPEND=""
 
 src_prepare() {
 	default
-	sed -i \
-		-e '/^CFLAGS/d' \
-		Makefile || die "sed failed"
+	sed -i -e '/^CFLAGS/d' Makefile
```

Comment:
No, the ```die```command must always stay, since ```sed``` is an external command. I only meant to drop the comment after the die commands :-)

Reply:
Oh, after your comment I thought ebuilds use -e mode implicitly nowadays :)

Reply:
Lgfm. But please squash your sed commit with the version bump, so you will have 2 commits again.